doorkeeper-orms.gemspec

```diff
@@ -10,17 +10,23 @@ Gem::Specification.new do |s|
   s.authors     = ["jasl"]
   s.email       = ["jasl9187@hotmail.com"]
   s.homepage    = "http://github.com/doorkeeper-gem/doorkeeper-orms"
-  s.summary     = "Doorkeeper with extracted ORM specifics, including mongoid 2-4 and mongo_mapper."
-  s.description = "Doorkeeper with extracted ORM specifics, including mongoid 2-4 and mongo_mapper"
+  s.summary     = "Doorkeeper mongoid 2, 3, 4 and mongo_mapper ORMs"
+  s.description = "Doorkeeper mongoid 2, 3, 4 and mongo_mapper ORMs"
```

Comment:
This is the critical problem.

You may found [specs about models](https://github.com/doorkeeper-gem/doorkeeper/tree/master/spec/models/doorkeeper), by theory you implement a ORM support, and pass specs for your job, that's enough to proving the correctness. but for Doorkeeper it's can not.

As my opinion, I refactor models and extract ORM support, but that's not enough, we don't have a obvious interfaces for models to tell how to play with them, so developers can not know what they need to implement and what the behaviour are. this point also discussion in PR #281.

so I think the critical thing is define these interfaces and those behaviours, and we could make a suppose that if a developer implement those correctly, the doorkeeper will work correctly.